components/GameDetailsView.jsx

```diff
@@ -169,6 +170,10 @@ const StyledArticle = styled.article`
     border-radius: var(--borderRadius);
     box-shadow: var(--boxShadow);
   }
+
+  @media screen and (min-width: 1024px) {
+    max-width: 1144px;
+  }
```

Comment:
🤔 On a screen that is at least 1024 px wide, the max-width should be 1144px? 

```suggestion
max-width: 1144px;
```

Reply:
1024px is the breakpoint after which the StyledArticle should be limited in width to 1144px. Since 1024px is the common breakpoint for all other media querys concerning the desktop view its also used here